common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -20,7 +18,7 @@
               % else:
               ${label}
               %endif
-          </span> - ${status.display_name}
+          </span> ${status.display_name}
```

Comment:
@ahsan-ul-haq I can see the status text here appear in the lower right corner of this section.  I would move the span after the status so that it is contained in the ```sr``` span.